torch/_inductor/fx_passes/pre_grad.py

```diff
@@ -80,7 +80,7 @@ def lazy_init():
         from . import fb  # type: ignore[attr-defined]  # noqa: F401
 
 
-def pre_grad_passes(gm: torch.fx.GraphModule, example_inputs):
+def pre_grad_passes(gm: torch.fx.GraphModule, example_inputs=None):
```

Comment:
Did it not work to construct the example_inputs from tensor meta? what were the failures ?

Reply:
`tensor_meta` doesn't include device. There is a device check here in the `pre_grad.py`:

https://github.com/pytorch/pytorch/blob/5b5bcf0470692c3bfb06b0459671b47f0c987f9e/torch/_inductor/fx_passes/pre_grad.py#L176

And we can't really extrapolate devices from the available `example_inputs` to the parent graph to the subgraph inputs.